bin/ledger2beancount-txns

```diff
@@ -130,14 +133,30 @@ sub pp_metadata($$) {
     return "$key: $value";
 }
 
+
+# determine whether something should be a link or a tag
+sub tag_or_link(@) {
+    my ($key) = @_;
+
+    foreach my $link_RE (@{$config->{"tag-is-link"}}) {
```

Comment:
All other configuration keys use `foo_bar_baz` convention, while some of those introduced in this PR use `foo-bar-baz`. We should be consistent, and `foo_bar_baz` is superior as it requires less often quoting, e.g., if we change conf file format.
(The change should also be ported to the documentation.)